### ISSUE TITLE
Sniper Elite 3 (238090)

### ISSUE THREAD

**Issue by ghost** [ValveSoftware/Proton#1152]
https://gist.github.com/daasuharo/d83c92ac04a9bd0c5edfc359c3c5a4de
Proton 3.7-5
Two error messages

error -- path not found
error launching the game: (0x000000050) - Access denied.

**Comment by r4mp** [comment]
Maybe the reason for the error is the same as described here https://steamcommunity.com/app/238090/discussions/1/458604254427614083

**Comment by Simmonz** [comment]
I have this same issue as well with the same error messages. I have attached a log file:

[steam-238090.log](https://github.com/ValveSoftware/Proton/files/2897002/steam-238090.log)

System Information:

OS: Manjaro

GPU: RX 580 8GB

Driver/LLVM version: Mesa 18.3.3

Kernel version: 4.20.11-1.-MANJARO

Proton version: 3.16-4

https://gist.github.com/Simmonz/7a6eb7ee79fcb5deee2a12633f5d8a58

**Comment by DrewBarrett** [comment]
> Maybe the reason for the error is the same as described here https://steamcommunity.com/app/238090/discussions/1/458604254427614083

Yep, no error windows once you move the library path, however the game never launches. (Runs for about 10 seconds then exits, no window or anything)

[steam-238090.log](https://gist.github.com/DrewBarrett/5d84612c24149205710643c9c908e231)
